### PR TITLE
Considering the text I think you are right. it should be in the 2019 

### PR DESCRIPTION
**Pull request by j0rt3g4**
Solving Issue #1142

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by JohanFreelancer9** [comment]
Merged before copy review was done.

**Comment by j0rt3g4** [comment]
Please proceed with thew copy review

File: SharePoint/SharePointServer/administration/upgrade-sharepoint-2016-to-sharepoint-2019-through-workflow-manager.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ SharePoint stores the workflow history and workflow task information for SharePo
 
 ### Prerequisites
 
-The following prrequisites must be done installed this upgrade:
+The following prequisites must be done installed this upgrade:
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/administration/upgrade-sharepoint-2016-to-sharepoint-2019-through-workflow-manager.md]: The following prerequisites must be met before installing this upgrade.

Comment (JohanFreelancer9) [reply SharePoint/SharePointServer/administration/upgrade-sharepoint-2016-to-sharepoint-2019-through-workflow-manager.md]: Please fix typo (**prerequisites**) and other suggestion to clarify the sentence.

Comment (j0rt3g4) [reply SharePoint/SharePointServer/administration/upgrade-sharepoint-2016-to-sharepoint-2019-through-workflow-manager.md]: Fixes on PR #1161

**Comment by j0rt3g4** [comment]
@scanum  continues on PR #1161

**Pull request merged**